File: submissions/sm_034_Venkat/week_3/day_4/session_1/mirrorImage.js
function strg(string) {
    var string = string || 'AyzB';
    var c = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
    var s = 'abcdefghijklmnopqrstuvwxyz';
    var str = ''
    for (i = 0; i < string.length; i++) {
        for (j = 0; j < c.length; j++) {
            if (string[i] === c[j]) {
                str = str + s[25 - j]
            }
            else if (string[i] === s[j]) {
                str = str + c[25 - j]
            }
        }
    }
    console.log(str)
}
strg('AvYfkjGR')

File: submissions/sm_112_soumik/week_13/day_4/session_1/mirror_image.py
strng = "Ayzb"
# # find mirror
string1 = "abcdefghijklmnopqrstuvwxyz"
string2 = string1.upper()
mirror = ""

for char in strng:
    for i in range(0, len(string1)):
        if char == string1[i]:
            mirror += string1[25 - string1.index(string1[i])]
        elif char == string2[i]:
            mirror += string2[25 - string2.index(string2[i])]
print(mirror)
